app/helpers/messages_helper.rb

```diff
@@ -0,0 +1,13 @@
+module MessagesHelper
+  def alert_class(alert)
+    if alert == 'error' || alert == 'alert'
+      'alert-danger'
+    elsif alert == 'success'
+      'alert-success'
+    elsif alert == 'notice'
+      'alert-info'
+    else
+      'alert-info'
+    end
+  end
+end
```

Comment:
@mllocs maybe we can move this to `ApplicationHelper`, I think we don't need an extra module only for this helper method (I'm also removing "blank" helpers in #361).

Reply:
I'll create an issue where we can discuss this, I prefer to tackle it in a different PR since we already tested it 😄

Reply:
no prob @mllocs, I'll do it in #361 while rebasing develop